cert/cert.go

```diff
@@ -464,7 +463,10 @@ func (spec *Spec) IsChangedOnDisk(path string) bool {
 
 // Reset the lifespan to force cfssl to regenerate
 func (spec *Spec) ResetLifespan() {
-	spec.tr.Provider.Certificate().NotAfter = time.Time{}
+	cert := spec.tr.Provider.Certificate()
+	if cert != nil {
```

Comment:
Is this an error if cert == nil?

Reply:
Not necesarily- the content may not exist on disk, thus Certificate() returns nil.